microsoft-edge/web-platform/release-notes/135.md

```diff
@@ -0,0 +1,462 @@
+
+* Minor discrepancies: Although there are slight differences in how the two limits are calculated, these differences are minor, and can be effectively managed within the `maxInterStageShaderVariables` limit.
+
+* Simplification: Removing `maxInterStageShaderComponents` streamlines the shader interface, and reduces complexity of using the interface.  Instead of managing two separate limits (that both apply simultaneously, but with subtle differences), you can just concentrate on the more appropriately named and comprehensive `maxInterStageShaderVariables`.
+
+
+<!-- ====================================================================== -->
+## Origin trials
+
+The following are new experimental APIs which you can try on your own live website for a limited time.
+
+To learn more about origin trials, see [Use origin trials in Microsoft Edge](../../origin-trials/index.md).
+
+To see the full list of available origin trials, see [Microsoft Edge Origin Trials](https://developer.microsoft.com/microsoft-edge/origin-trials/).
+
+
+<!-- ------------------------------ -->
+#### Microsoft Edge-only origin trials
+
+
+<!-- ---------- -->
+###### Digital Goods API
+
+Expires on March 31, 2025.
+
+The Digital Goods API allows a web application to get information about their digital products and the user’s purchases managed by a digital store.  The user agent abstracts connections to the store, and the Payment Request API is used to make purchases.
+
+* [Explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/PwaDigitalGoods/explainer.md)
+* [Feedback](https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/new?labels=Digital+Goods%20API,OriginTrialFeedback&amp;title=%5BDigital%20Goods%20API%5D+Feedback)
+* [Register](https://developer.microsoft.com/en-us/microsoft-edge/origin-trials/trials/4b4a9ead-d912-4349-87b3-25e5e50b4f13)
+
+
+
+<!-- ---------- -->
+###### Web App LocalFolder Access
+
+Expires on March 31, 2025.
+
+Web App LocalFolder Access allows a Microsoft Store-installed Progressive Web App (PWA) to access file content that was previously stored in the WinRT `ApplicationData.LocalFolder` folder by an earlier,<!-- todo: analyze implication of comma vs no comma --> UWP version of the application.
+
+* [Explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/LocalFolder/explainer.md)
+* [Feedback](https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/new?labels=LocalFolder+Access,OriginTrialFeedback&amp;title=%5BLocalFolder%20Access%5D+Feedback)
+* [Register](https://developer.microsoft.com/en-us/microsoft-edge/origin-trials/trials/2b21a8cd-43aa-41da-810d-fad103e699dc)
+
+
+
+<!-- ---------- -->
+###### HTML+IDL handwriting attribute
+
+Expires on June 14, 2025.
+
+Granular per-document and per-element control over which content should (dis)allow<!-- todo: ungrammatical, confusing, clarify --> handwriting input.
```

Comment:
do not use:   (dis)allow

rewrite in plain explicit direct language to explain how it works, using dictionary words & grammar.